webnode/config/webpack.config.js

```diff
@@ -76,7 +76,8 @@ const common = {
               compact: true,
               presets: ["react-app"],
               plugins: [
-                "transform-object-rest-spread"
+                "transform-object-rest-spread",
+                "lodash"
```

Comment:
I think we removed lodash, didn't we?  Or maybe we just used a plugin to make it smaller?

Reply:
@rfornea we still have lodash, i tried to use a plugin but it ended up being more trouble than it was worth.

@tenisakb either there is a merge available or there was a bad merge. the lodash plugin was removed.

Reply:
Okay nvm about lodash Ladislav.  Just keep it.